Vagrantfile

```
@@ -52,11 +52,12 @@ VERSION=$(lsb_release -sr | tr '[:upper:]' '[:lower:]')
 bash ${GIT_PATH}/.setup/install_worker.sh #{extra_command} 2>&1 | tee ${GIT_PATH}/.vagrant/install_worker.log
 SCRIPT
 
-box_extra = if Vagrant::Util::Platform.darwin? && (`uname -m`.chomp == "arm64" || (`sysctl -n machdep.cpu.brand_string`.chomp.include? 'M1'))
-  '-arm64'
-else
-  ''
-end
+base_boxes = Hash[]
+
+# Should all be base Ubuntu 20.04 boxes
+base_boxes.default         = "bento/ubuntu-20.04"
+base_boxes[:arm_parallels] = "bento/ubuntu-20.04-arm64"
+base_boxes[:libvirt]       = "generic/2004"
```

I think there's a typo on this line?
```suggestion
base_boxes[:libvirt]       = "generic/ubuntu2004"
```